deployment/deployment.go

```diff
@@ -280,6 +297,15 @@ func getDeploymentSelectionNamespaces(client *houston.Client, out io.Writer) (st
 	return names[i-1].Name, nil
 }
 
+func getDeploymentNamespaceName() (string, error) {
+	namespaceName := input.Text("\nKubernetes Namespace Name: ")
+	noSpaceString := strings.ReplaceAll(namespaceName, " ", "")
+	if noSpaceString == "" {
```

Comment:
Does it make sense to do empty string validation?

Reply:
It does make sense to me to validate the input, but it would also be decent idea to strip unwanted spaces from the input text entered by the user, i.e.
```
namespaceName = strings.ReplaceAll(namespace, " ", "")
if namespace == "" {
    return "", ErrKubernetesNamespaceNotSpecified
}
return namespaceName, nil
```